org.eclipse.xtext.common.types.eclipse.tests/tests/org/eclipse/xtext/common/types/access/xtext/ui/JdtBasedConstructorScopeTest.java

```diff
@@ -73,7 +73,7 @@ public boolean apply(IEObjectDescription input) {
 		assertFalse(Iterables.any(contents, new Predicate<IEObjectDescription>() {
```

Comment:
should be assertTrue i thing. @szarnekow what do you think?

Reply:
but i am not sure here, cause default constructor is protected

Reply:
Having debugged the test I can say that AbstractList is present, I think this should be assertTrue as well.